package.json

```diff
@@ -21,6 +21,8 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "@wormhole-foundation/sdk-base": "^0.6.5",
```

Comment:
The `builder` depends on `@wormhole-foundation/wormhole-connect@0.3.3` which depends on an older version of `sdk-base` that doesn't contain the definitions for Blast. The older version of `sdk-base` gets hoisted to the `node_modules` in the root directory, so installing the latest version of `sdk-base` and `sdk-definitions` here overrides that and works. This was the simplest solution I found to work.